src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Sse2.PlatformNotSupported.cs

```diff
@@ -774,15 +774,13 @@ public new abstract class X64 : Sse.X64
         public static unsafe Vector128<double> LoadLow(Vector128<double> upper, double* address) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128i _mm_loadl_epi32 (__m128i const* mem_addr)
+        /// __m128i _mm_loadu_si32 (void const* mem_addr)
         ///   MOVD xmm, reg/m64
```

Comment:
While you're in there, these should also be `m32` 😄

Reply:
Fixed. Looks like this one was just in the PNSE version of the file.